redisinsight/api/src/modules/rdi/client/rdi.client.ts

```diff
@@ -23,6 +23,12 @@ export abstract class RdiClient {
 
   abstract getPipeline(): Promise<RdiPipeline>;
 
+  // TODO validate options and response
+  abstract getTemplate(options: object): Promise<unknown>;
+
+  // TODO validate response schema
```

Comment:
Is this going to be completed in a separate card?

Reply:
same